config.yaml

```diff
@@ -54,6 +54,14 @@ core_config_version: 0
 # (OPTIONAL | Default: 900000) long value. Time in milliseconds for how long a passwordless code is valid for.
 # passwordless_code_lifetime:
 
+# (OPTIONAL | Default: 5) integer value. The maximum number of invalid TOTP attempts that will trigger rate limiting.
+# totp_max_attempts:
+
+# (OPTIONAL | Default: 900) integer value. The time in seconds for which the user will be rate limited once totp_max_attempts is crossed.
+# totp_rate_limit_cooldown_sec:
```

Comment:
Hmm. We can use 5 mins.

Here's how we can measure it:
- There are 1 million possible codes in any period (because of 6 digits)
- But because of skew, we check for a total of `(2*skew+1)` codes (not just one) 
- `period_valid_codes = (2 * skew + 1)`
- `attempts_per_hour = (60 / cooldown_mins * period_valid_codes)`
- `1 - binom.pmf(k=0, n=attempts_per_hour * hours, p=period_valid_codes / 10_00_000)` gives the probability that an attacker will be successful at least once. 

So given a cooldown of 5 mins and skew of 5 (i.e. 11 valid codes for each period), it will take 3200 hours (~4.5 months) to break TOTP with a 50% chance which makes the attack infeasible.